toonz/sources/toonz/sceneviewerevents.cpp

```diff
@@ -307,20 +308,13 @@ void SceneViewer::tabletEvent(QTabletEvent *e) {
 #endif
   } break;
   case QEvent::TabletMove: {
-#ifdef MACOSX
     // for now OSX seems to fail to call enter/leaveEvent properly while
     // the tablet is floating
     bool isHoveringInsideViewer =
         !rect().marginsRemoved(QMargins(5, 5, 5, 5)).contains(e->pos());
     // call the fake enter event
     if (isHoveringInsideViewer) onEnter();
-#else
-    // for Windowsm, use tabletEvent only for the left Button
-    if (m_tabletState != StartStroke && m_tabletState != OnStroke) {
-      m_tabletEvent = false;
-      break;
-    }
-#endif
```

Comment:
Was there a reason this conditional macro was removed? I'm concerned this will bring back an issue that was previously resolved and/or cause unwanted side effects for Windows users.